install/kubernetes/cilium/charts/agent/templates/daemonset.yaml

```diff
@@ -29,10 +29,48 @@ spec:
 {{ toYaml .Values.global.affinity | indent 8 }}
 {{- end }}
       containers:
+{{- if .Values.global.holdInitAfterClean }}
```

Comment:
A comment why this is used would be helpful.

Reply:
@brb we usually have comments in the charts/value.yaml not here.